filter2-monica.js

```diff
@@ -1,6 +1,8 @@
 var misc = [3, "true", {a: 1, b: 2}, 7, [1, 2, 3], ['a', 'b', 'c'], "my favorite string", 59.09]
 
-// Use filter to filter all strings out of the above array. 
+// Use filter to filter all strings out of the above array.
 // Don't worry about strings in the nested array (misc[5] is considered an array, not a string)
 
 // Your code goes here
+const stringsOnly = misc.filter(elem => typeof elem === "string")
+console.log(stringsOnly)
```

Comment:
you filtered out all the things that *aren't* strings instead of all the things that *are* strings 😂 no worries, though, that's a simple character replacement in your equality operator. regardless, though, nice job with this! 👍